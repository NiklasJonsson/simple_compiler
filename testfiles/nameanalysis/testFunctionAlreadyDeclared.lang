int fun() {
	int a;
}
int fun() {
	int b;
}