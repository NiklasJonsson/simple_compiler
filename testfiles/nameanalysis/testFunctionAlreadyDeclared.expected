Error at line 4: symbol 'fun' is already declared!
